corehq/apps/app_manager/templates/app_manager/partials/filter_configs.html

```diff
@@ -17,37 +17,51 @@
                     value: selectedValue.doc_type
                 ">
                 </select>
-            </th>
-            <th>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'AutoFilter'">
-                    <select data-bind="
-                        options: report.filterConfig.autoFilterTypes,
-                        value: filter.selectedValue.filter_type
-                    "></select>
-                </div>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticDatespanFilter'">
-                    <select data-bind="
-                        options: report.filterConfig.date_range_options,
-                        value: filter.selectedValue.date_range
-                    "></select>
-                </div>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'CustomDataAutoFilter'">
-                    <input type="text" data-bind="value: filter.selectedValue.custom_data_property"/>
-                </div>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticChoiceListFilter'">
-                    <input type="text"
-                           class="choice_filter"
-                           style="width:300px"
-                           data-bind="value: filter.selectedValue.value"/>
-                    <input type="hidden" data-bind="value: dynamicFilterName">
-                </div>
-                <div data-bind="visible: filter.selectedValue.doc_type() === 'StaticChoiceFilter'">
-                    <select data-bind="
-                        options: filter.choices,
-                        optionsText: 'display',
-                        optionsValue: 'value',
-                        value: filter.selectedValue.select_value
-                    "></select>
+                <div style="padding-top:10px">
```

Comment:
I poked at your branch for a while trying to find a way to get rid of this wrapper div. Turns out this whole reports area is inside of a `form-horizontal`, which kills the `margin-bottom` that inputs and selects normally get. The reports area should be on a separate tab from the module settings, but that's not really within the scope of this PR. Inline styling is bad but does seem like the best option here.